CMakeLists.txt

```diff
@@ -294,8 +294,12 @@ endif()
 if (ONUT_USE_SDL)
     set(HAVE_LIBC ON)
     set(SDL_MAIN_HANDLED ON)
+    if (WINDOWS)
     add_subdirectory(./thirdparty/SDL ./thirdparties/SDL)
     list(APPEND libs SDL2-static)
+    else()
+    list(APPEND libs SDL2)
```

Comment:
Had compile issues with the SDL2 submodule, but doesnt really matter because most linux systems ship their own shared lib of SDL2 anyway.